### ISSUE TITLE
Aurora becomes incredibly slow and unresponsive (no crash or application not responding)

### ISSUE THREAD

**Issue by undermark5** [antonpup/Aurora#2030]
### Expected Behaviour
Aurora should function and be responsive

### Actual Behaviour
Aurora will become slow and completely unresponsive (but Windows doesn't say that the application is not responding) until ending the task and restarting it.

### Reproduction steps
I can always reproduce by just running the thing and waiting (doesn't matter if the computer goes to sleep or not) I have included DMP files in [a zip](https://drive.google.com/open?id=1mgxOBegJZrOPGbgUVj5yqufmpj6iE99N), they are from different times of starting the application and then taken when it became slow and unresponsive.

### Frequency
Every 

time eventually, though the time it takes is different

### Aurora Version:
beta v0.7.2+799

### Previously an Issue?
also occurred on latest stable

**Comment by Bluscream** [comment]
Prbobly bad pc?

**Comment by undermark5** [comment]
Umm, most certainly not bad PC... I have Intel i5-7600k 16GB of RAM and RX 580 I'm using Aurora with the vanguard k08. This occurs regardless of what else is running on the system.

**Comment by xQwexx** [comment]
It is probably cased by same of the rgb device, which is not updating are to slow. 
Try disable them by replace the device_disabled in the Appdata/Roaming/Aurora/Config.json
```
  "devices_disabled": {
    "$type": "System.Collections.Generic.HashSet`1[[System.Type, mscorlib]], System.Core",
    "$values": [
      "Aurora.Devices.Dualshock.DualshockDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.UnifiedHID.UnifiedHIDDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.NZXT.NZXTDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Wooting.WootingDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Logitech.LogitechDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Corsair.CorsairDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Razer.RazerDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Roccat.RoccatDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Clevo.ClevoDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.CoolerMaster.CoolerMasterDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.SteelSeries.SteelSeriesDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.AtmoOrbDevice.AtmoOrbDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Creative.SoundBlasterXDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.LightFX.LightFxDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.YeeLight.YeeLightDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Drevo.DrevoDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Asus.AsusDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Vulcan.VulcanDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Uniwill.UniwillDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
      "Aurora.Devices.Ducky.DuckyDevice, Aurora, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
    ]
  }
```

**Comment by undermark5** [comment]
@xQwexx , well looks like if you actually read any of the comments you would realize that your json would disable the keyboard that I have... obviously this isn't that difficult of a fix, but it shows that you didn't really take the time to provide an accurate answer... Also, the uniwill device is not valid at least for the version that I'm running... And finally even after fixing these errors of yours, adding those devices to the disabled list still does not solve the problem. 

And since you asked, here is a log file from when this issue happened, I doubt it is helpful in any way shape or form though because the error happened at 16:53 and last lines in the log file are from 16:49...
[2020-05-22 16.48.59.log](https://github.com/antonpup/Aurora/files/4670566/2020-05-22.16.48.59.log)

**Comment by diogotr7** [comment]
The whole point of disabling **all** of the devices was seeing if they were causing the problem, or if it was something else. By keeping the deviuce integration that works with your keyboard on, we're back to stage 0 in figuring out where the issue comes from.

Do you have any other devices besides the keyboard that could be possibly causing this?
Which layers are you using? Do you play aurora supported games?

**Comment by undermark5** [comment]
Right, no, I do not have any other devices (at least not that I'm aware). The layer that I think may be causing the issue is the shortcut layers that were there by default under the desktop settings. The issue seems to arise when there are a large number of key presses , and I just tried deleting those layers entirely (leaving just a solid base layer) and after doing a 5 minute online typing test, the issue was still present. I play factorio, but the issue will happen at any time regardless of if the desktop profile is active or the game profile is active.
